app/helpers/form_inputs_helper.rb

```diff
@@ -25,12 +25,11 @@ def education
 end
 
 def pref_language
-	[
-		['English', 'English'], 
-		['Spanish', 'Spanish'], 
-		['Polish', 'Polish'],
-		['Other', 'Other']
-	]
+  if I18n.locale != I18n.default_locale
+    [%w[Inglés English], %w[Español Spanish], %w[Polaco Polish], %w[Otro Other]]
+  else
+    [%w[English English], %w[Spanish Spanish], %w[Polish Polish], %w[Other Other]]
+  end
```

Comment:
that `%w` is just a notation to write an array of strings separated by spaces instead of commas and without quotes around them. I guess this is considered better practice according to the rails style guide.